pandas/tests/plotting/test_frame.py

```diff
@@ -1054,17 +1050,15 @@ def test_plot_scatter_with_c(self):
                        index=list(string.ascii_letters[:6]),
                        columns=['x', 'y', 'z', 'four'])
 
-        axes = [df.plot.scatter(x='x', y='y', c='z'),
-                df.plot.scatter(x=0, y=1, c=2)]
-        for ax in axes:
-            # default to Greys
-            assert ax.collections[0].cmap.name == 'Greys'
+        axes = df.plot.scatter(x='x', y='y', c='z')
 
-            if self.mpl_ge_1_3_1:
+        # default to Greys
+        assert axes.collections[0].cmap.name == 'Greys'
 
-                # n.b. there appears to be no public method to get the colorbar
-                # label
-                assert ax.collections[0].colorbar._label == 'z'
+        if self.mpl_ge_1_3_1:
```

Comment:
@TomAugspurger we should be able to blow this code away as we > 1.4.3, yes (separate PR)